README.rst

```diff
@@ -36,6 +36,19 @@ To change the social icons, edit ``mpl_sphinx_theme/mpl_icon_links.html``
 
 To change the style, edit ``mpl_sphinx_theme/static/css/style.css``
 
+Overriding hard coded elements
+------------------------------
+This theme is designed to be used with subprojects that are part of the main
```

Comment:
```suggestion
This theme is primarily designed to be used with subprojects that are part of the main
```

Reply:
I found the absolute character of the original wording slightly confusing. You say it's designed for matplotlib subprojects, but there are also other subprojects.